pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -136,6 +136,7 @@ mkCommon type rec {
 
   passthru = {
     inherit icu;
+    packages_list = packages;
```

Comment:
Wasn't sure how to get the list of projects so I just readded this since the previous pr did that. If there's a better way do get the packages already retrieved from dotnet, then we can change this.